Guidelet/GuideletLib/Guidelet.py

```diff
@@ -38,16 +38,16 @@ def onGenericCommandResponseReceived(commandId, responseNode):
       logging.debug("Timeout. Command Id: {0}".format(commandId))
 
   # Guidelet layout name definitions
-  VIEW_ULTRASOUND = unicode("Ultrasound")
-  VIEW_ULTRASOUND_3D = unicode("Ultrasound + 3D")
-  VIEW_3D_ULTRASOUND = unicode("3D + Ultrasound")
-  VIEW_ULTRASOUND_CAM_3D = unicode("Ultrasound + Webcam + 3D")
-  VIEW_ULTRASOUND_DUAL_3D = unicode("Ultrasound + Dual 3D")
-  VIEW_3D = unicode("3D")
-  VIEW_DUAL_3D = unicode("Dual 3D")
-  VIEW_TRIPLE_3D = unicode("Triple 3D")
-  VIEW_TRIPLE_3D_PARALLEL = unicode("Triple 3D Parallel")
-  VIEW_QUAD_3D = unicode("Quad 3D")
+  VIEW_ULTRASOUND = ("Ultrasound").encode
```

Comment:
I think you mean ("").encode()

Reply:
Good catch. Although I think instead of using encode I will alias str as unicode for Python 3.